content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,90 @@
+# Continuous Integration and Deployment for Machine Learning
+
+## DevOps
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers), testing, and deployment to production (CI/CD --
+continuous integration and continuous delivery), as well as monitoring &
```

Comment:
Minor. But could cause confusion. Should we stick to D = deployment, as in the title?

Reply:
the official definition of CI/CD uses "delivery."